### ISSUE TITLE
[BUG] Web: URL containing '+' character is rejected by IIS reverse proxy

### ISSUE THREAD

**Issue by usikpavel** [immich-app/immich#635]
I'm using an IIS as reverse proxy for redirecting requests to Immich-server. IIS has a default rejecting behaviour (with HTTP Error 404.11) for `+` character described here: [IIS7 rejecting URLs containing +](https://blogs.iis.net/thomad/iis7-rejecting-urls-containing)

Currently, I found only one problem URL: `/_app/immutable/assets/+layout-c9d87204.css`. I'm allowing this URL manually by configuring `<alwaysAllowedUrls>` option, but file name is changing in every build, and every time Immich-web has updated I need update this option.

**Expected behavior**
Remove `+` character from `+layout-c9d87204.css` filename or replace it with some valid character.

**System**
- Server Version: `v1.28.0`

**Comment by bo0tzz** [comment]
Can you not turn off this filter feature for the entire Immich proxy?

**Comment by usikpavel** [comment]
Yes, I can totally disable this filter by `allowDoubleEscaping` option. But there are security reasons why this behaviour was introduced. I'm not an IB specialist, I only can give a following explanation: [https://stackoverflow.com/questions/1453218/is-enabling-double-escaping-dangerous#comment1300728_1453287](https://stackoverflow.com/questions/1453218/is-enabling-double-escaping-dangerous#comment1300728_1453287)

**Comment by alextran1502** [comment]
The `+` symbol is the convention of the web framework. There is not a way to actually remove it

**Comment by usikpavel** [comment]
Ок, understand.
